features/default/configuration.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Configuration overrides
+
+  Scenario: Override service name
+    Given I run "ServiceNameScenario"
```

Comment:
Can this test be generalised slightly to assert the override of bundle version and service version as well? (I appreciate that hasn't changed, but it feels like a natural place given the feature name and we don't appear to be testing this elsewhere yet.)